src/algebra/ring/boolean_ring.lean

```diff
@@ -168,3 +169,168 @@ localized "attribute [instance, priority 100] boolean_ring.to_boolean_algebra" i
   boolean_algebra_of_boolean_ring
 
 end boolean_ring
+
```

Comment:
The stuff in this file could be generalized to `generalized_boolean_algebra`s and `nonunital_ring`s, but the latter doesn't exist (yet?).